rerun_cpp/tests/recording_stream.cpp

```diff
@@ -157,6 +157,11 @@ void test_logging_to_file(const char* test_rrd, rr::RecordingStream& stream) {
         }
 
         // The file already has some header from the start.
+        //
+        // ATTENTION:
+        // Particularly on MacOS file size checks like this are often unreliable as they tend to not
+        // read the latest state on the file. I have not observed any failure yet, but if this test
+        // ever gets flaky, this is the first place to look.
```

Comment:
Well I guess that explains things ^_^

We should definitely not be relying on filesystem metadata, people have all kinds of crazy mount options on Linux.

Just read the file and count the bytes the old way?

Reply:
let's try that before putting in sleepenings, yes